### ISSUE TITLE
Build shortcode to display any post type in any layout

### ISSUE THREAD

**Issue by juliotaylor** [bigspring/monolith#390]
A shortcode that can render any post type, in any layout, in any order, for example:

'[build post-type="products" layout="snippets" orderby="date" order="asc"]'

**Comment by juliotaylor** [comment]
@jrgmartin for review / further detail.

Could we refactor all shortcodes into one single one to do this?

**Comment by harrybigspring** [comment]
Could this also include case studies, testimonials and single versions?

'[build post-type="testimonial" layout="blockquote" id="23"]'

'[build post-type="testimonial" layout="blockquote" category="dogs" orderby="date" order="asc"]'

**Comment by juliotaylor** [comment]
well, the problem with category is that it's a dynamic taxonomy, which could change from project to project.

It would need a 'taxonomy' variable too.

@jrgmartin

**Comment by jrgmartin** [comment]
Yes, this is totally do-able and relatively easy to implement.  

For taxonomy, we would actually need two vars: 
1. Taxonomy
2. Term

**Comment by juliotaylor** [comment]
Can you build this into the spec for the new builder tools in M3?
